### PR TITLE
refactor: client

### PR DESCRIPTION
**Pull request by anshumanv**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [x] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [ ] This is a **metadata update**

### For Bugs and Features; did you add new tests?

existing

### Motivation / Use-Case

Closing stale PR https://github.com/webpack/webpack-dev-server/pull/2431

### Breaking Changes

no

### Additional Info

Close https://github.com/webpack/webpack-dev-server/pull/2431

### CODE REVIEW

**Review by snitin315 — APPROVED** [review]

**Comment by anshumanv** [comment]
There's one more instance https://github.com/webpack/webpack-dev-server/blob/84e1d3426684849caca2104f834e4c83d86b1e17/lib/Server.js#L1018

But I feel it's better as is

File: client-src/index.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ self.addEventListener('beforeunload', () => {
 if (typeof window !== 'undefined') {
   const qs = window.location.search.toLowerCase();
 
-  options.hotReload = qs.indexOf('hotreload=false') === -1;
+  options.hotReload = !qs.includes('hotreload=false');
```
Comment (alexander-akait) [inline client-src/index.js:69]: Don't no use modern function in client due some developers still need IE 11 support and other old browsers https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Comment (anshumanv) [reply client-src/index.js:69]: but we transpile them using babel right?

Comment (alexander-akait) [reply client-src/index.js:69]: Yes, but it is increase our bundle, ideally we should keep this small as possible

Comment (anshumanv) [reply client-src/index.js:69]: gotcha lets change this then

Comment (anshumanv) [reply client-src/index.js:69]: startsWith will also come after transpile, remove it too? I think it won't affect the bundle that much as polyfill will be really small..

Comment (alexander-akait) [reply client-src/index.js:69]: Yes, they are small, but why do this if we can use alternatives and not increase the size at all

Comment (anshumanv) [reply client-src/index.js:69]: makes sense, should I leave change in `Server.js` or close?

**Comment by alexander-akait** [comment]
Done in master

**Pull request closed without merging**